app/pages/creator-landing-page/[pubKey].tsx

```diff
@@ -51,7 +72,12 @@ const CreatorLandingPage = () => {
     );
     console.log(creatorPubKey.toBase58());
 
-    const creatorAccount = await program!.account.creator.fetch(creatorPubKey);
+    // route to different page if creator is not found from pubkey, create a new component from it
+    const creatorAccount = await ( pubKey == creatorPubKey.toBase58() ? program!.account.creator.fetch(pubKey) : null);
+    if (creatorAccount == null) {
+      router.replace('/creator-landing-page/creator-not-found');
```

Comment:
We don't need a separate route for this. You can use a `boolean` state value and do something like this in this component:
```js
if (creatorNotFound) return (
  <div>
    ...
  </div>
)
```